etc/HIProdOfflineConfiguration.py

```diff
@@ -1429,404 +1529,295 @@
+DATASETS = ["HIHLTPhysics"]
 
-for dataset in datasets:
+for dataset in DATASETS:
     addDataset(tier0Config, dataset,
-               write_miniaod = False,
-               do_reco = True,
-               write_dqm = False,
-               dqm_sequences = [ "@none" ],
-               scenario = hiTestppScenario)
+               write_miniaod=False,
+               do_reco=True,
+               raw_to_disk=True,
+               write_dqm=True,
+               alca_producers=["TkAlMinBias"],
+               dqm_sequences=["@common"],
+               scenario=hiTestppScenario)
 
-datasets = [ "HIHcalNZS" ]
+DATASETS = ["HIOnlineMonitor", "HITrackerNZS"]
 
-for dataset in datasets:
+for dataset in DATASETS:
     addDataset(tier0Config, dataset,
-               write_miniaod = False,
-               do_reco = True,
-               write_dqm = True,
-               alca_producers = [ "HcalCalMinBias" ],
-               dqm_sequences = [ "@common", "@hcal" ],
-               scenario = hcalnzsScenario)
+               do_reco=False,
+               raw_to_disk=True,
+               scenario=hiTestppScenario)
 
-datasets = [ "HIHLTPhysics" ]
+DATASETS = ["HIEmptyBX"]
 
-for dataset in datasets:
+for dataset in DATASETS:
     addDataset(tier0Config, dataset,
-               write_miniaod = False,
-               do_reco = True,
-               write_dqm = True,
-               alca_producers = [ "TkAlMinBias" ],
-               dqm_sequences = [ "@common" ],
-               scenario = hiTestppScenario)
+               do_reco=True,
+               write_dqm=True,
+               dqm_sequences=["@common"],
+               scenario=hiTestppScenario)
 
-datasets = [ "HIHardProbesLower" ]
+DATASETS = ["HITestRaw0", "HITestRaw1", "HITestRaw2", "HITestRaw3", "HITestRaw4", "HITestRaw5",
+            "HITestRaw6", "HITestRaw7", "HITestRaw8", "HITestRaw9", "HITestRaw10", "HITestRaw11",
+            "HITestRaw12", "HITestRaw13", "HITestRaw14", "HITestRaw15", "HITestRaw16", "HITestRaw17",
+            "HITestRaw18", "HITestRaw19", "HITestRaw20", "HITestRaw21", "HITestRaw22", "HITestRaw23"]
 
-for dataset in datasets:
+for dataset in DATASETS:
     addDataset(tier0Config, dataset,
-               write_miniaod = False,
-               do_reco = True,
-               write_dqm = True,
-               reco_split = hiRecoSplitting,
-               dqm_sequences = [ "@common", "@ecal", "@egamma" ],
-               scenario = hiTestppScenario)
+               write_miniaod=False,
+               do_reco=True,
+               raw_to_disk=True,
+               write_dqm=True,
+               alca_producers=["SiStripCalZeroBias", "SiStripCalMinBias","TkAlMinBias",
+                               "HcalCalIsolatedBunchSelector", "HcalCalIterativePhiSym","HcalCalMinBias",
+                               "TkAlJpsiMuMu", "TkAlUpsilonMuMu","TkAlZMuMu","TkAlMuonIsolated"],
+               dqm_sequences=["@commonSiStripZeroBias"],
```

Comment:
same comment here about the `dqm_sequences` added in [1c2f703](https://github.com/dmwm/T0/pull/4778/commits/1c2f703d2e8a1767fbdbec287737e9b828815421)

Reply:
done